editor/pnpm-lock.yaml

```diff
@@ -4640,7 +4640,7 @@ packages:
     dependencies:
       '@rollup/pluginutils': 3.1.0
       '@types/resolve': 1.17.1
-      builtin-modules: 3.2.0
+      builtin-modules: 3.3.0
```

Comment:
Looks like this was added in error?

Reply:
oh yes, looks like a leftover that didn't clean when I `pnpm uninstall`-ed it. thanks!